plugins/lando-core/tasks/destroy.js

```diff
@@ -20,7 +20,7 @@ module.exports = lando => {
       const app = lando.getApp(options._app.root);
       // Destroy the app
       if (app) {
-        console.log((chalk.green(`Preparing to resign ${app.name} to the dustbin of history...`)));
+        console.log((chalk.green(`Preparing to consign ${app.name} to the dustbin of history...`)));
```

Comment:
Actually, I think both verbs are fine.
```
resign: accept that something undesirable cannot be avoided.
consign: assign; commit decisively or permanently.
```

Reply:
No, that definition of resign only exists in the passive voice: for example, "I can't stand my cousin, but I _am resigned to_ her visit."
Consign implies a permanent and final disposition (source: https://www.merriam-webster.com/dictionary/consign, definition 2), which is what's being done with lando destroy. "Consign to the dustbin of history" also alludes to a specific quote of Leon Trotsky.

Reply:
But that is not what Trotsky quoted.  Here is what he said:
```
You are pitiful isolated individuals; you are bankrupts; your role is played out. Go where you belong from now on - into the dustbin of history!
```
What you argue is exactly why I hate the English language so much. :)

Reply:
If you'd quoted Trotsky in the Russian he actually spoke, I'd be more certain of your translation. Here's WIRED making use of the "consign" version, as one example: https://www.wired.com/2009/09/0904blackhole/.

And none of this contradicts my first two points, which are really the main ones. So why not quit your Googling and approve this? :)

Reply:
I'm not saying I disapprove.  I just think it's a petty difference not worth making a big fuss about. :)

Reply:
guys? WTF?

Reply:
The expression is "consign to the dustbin of history". If I get to the point where my app fails, I'm already stressed out, and seeing "resign" there just increses my stress. With thousands of users, I can't be the only one who feels this way. This change will make Lando look smarter.